src/components/search/curated/curated-essential.tsx

```diff
@@ -22,7 +22,7 @@ export type Topic = {
 type CuratedEssentialProps = {
   topic: Topic
   pageData?: any
-  CTAComponent?: React.FC
+  CTAComponent?: React.FC<any>
```

Comment:
TypeScript was complaining because the component being passed in is of type React.FC<{location: string}> not React.FC. Should this be updated to -- `React.FC<{location: string}>`?

Reply:
nah, you could have something like `CTACardType` if you wanted to make it good.